docs/dockerhub.md

```diff
@@ -0,0 +1,79 @@
+# jpo-sdw-depositor
+
+## GitHub Repository Link
+https://github.com/usdot-jpo-ode/jpo-sdw-depositor
+
+## Purpose
+The purpose of the jpo-sdw-depositor program is to deposit messages to the SDX.
+
+## How to pull the latest image
+The latest image can be pulled using the following command:
+> docker pull usdotjpoode/jpo-sdw-depositor:develop
+
+## Required environment variables
+- DOCKER_HOST_IP
+- SDW_EMAIL_LIST

```

Comment:
Do we really only require the email list and from? There isn't other info such as mail server endpoint and configuration info?

Reply:
The `DepositorProperties` class only checks for the emailList & emailFrom variables, so they are the only variables that are required to be set to get the image running.

The 'application.properties' file defines the following properties, but they don't appear to be referenced anywhere in the project.
- spring.mail.host
- spring.mail.port
- spring.mail.username
- spring.mail.password

The following environment variables are defined in the 'sample.env' file, but also do not appear to be used:
- SPRING_MAIL_HOST
- SPRING_MAIL_PORT

It may be worth looking into whether the email functionality for the project is working.

Reply:
The caps/snake messages are auto-translated into the dot notation of the other values, and because it's a spring level var it just gets auto-set by spring and we don't have to worry about it. Seems to be on the up and up.